src/annotator/guest.js

```diff
@@ -341,6 +341,16 @@ export default class Guest {
       }
     );
 
+    this._hostRPC.on(
+      'sidebarLayoutChanged',
+      /** @param {SidebarLayout} sidebarLayout */
+      sidebarLayout => {
+        if (this._frameIdentifier === null) {
```

Comment:
Reacting to the sidebar layout changes makes sense only to the `Guest` that is in the `host` frame. Currently, we know that's the case if `frameIdentifier` is `null`. In the future however, that can change, which will require adaptation.